### PR TITLE
Fix compilation error in preProcessMmf

### PR DESCRIPTION
**Pull request by singh264**
Define privatePortLibrary in preProcessMmf(..) for
defined(TR_TARGET_64BIT) && defined(J9ZOS390).

Signed-off-by: Amarpreet Singh amarpreet1997@gmail.com

### CODE REVIEW

File: runtime/compiler/control/J9Options.cpp [hunk 1]
```diff
@@ -1202,7 +1202,13 @@ static bool JITServerParseCommonOptions(J9JavaVM *vm, TR::CompilationInfo *compI
 
 void J9::Options::preProcessMmf(J9JavaVM *vm, J9JITConfig *jitConfig)
    {
+#if defined(TR_TARGET_64BIT) && defined(J9ZOS390)
+   PORT_ACCESS_FROM_JAVAVM(vm);
+   OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+#endif
```
Comment (dsouzai) [inline runtime/compiler/control/J9Options.cpp:1208]: Best to just move this down to where it's actually used, so we don't have two separate ifdefs.

Comment (singh264) [reply runtime/compiler/control/J9Options.cpp]: I have moved the changes to the below ifdef.

**Comment by singh264** [comment]
@dsouzai requesting feedback

**Review by dsouzai — APPROVED** [review]
LGTM

**Comment by dsouzai** [comment]
Not gonna run tests as this only changes code guarded by `defined(J9ZOS390)`. Awaiting results of internal z/OS build.

File: runtime/compiler/control/J9Options.cpp (hunk 1 above)
Comment (dsouzai) [reply runtime/compiler/control/J9Options.cpp]: Warnings aren't enabled on z/OS yet; the internal build succeeded.

**Pull request merged**